django/api/models/go_electric_rebate_application.py

```diff
@@ -126,6 +128,18 @@ def doc2_tag(self):
     consent_personal = BooleanField(validators=[validate_consent], null=True)
     consent_tax = BooleanField(validators=[validate_consent], null=True)
     reason_for_decline = CharField(max_length=500, unique=False, blank=True, null=True)
+    submission_date = DateTimeField(auto_now_add=True, null=True)
```

Comment:
Since GoElectricRebateApplication extends from TimeStampedModel, it already has a "created" field so we can just use that instead of creating a new submission_date field.